### PR TITLE
ddl: move some test to ddl_test package

### PR DESCRIPTION
**Pull request by xiongjiwei**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/34284

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: ddl/ddl_test.go [hunk 1]
```diff
@@ -16,14 +16,15 @@ package ddl
 
 import (
 	"context"
+	"fmt"
+	"github.com/pingcap/tidb/table/tables"
```
Comment (wjhuang2016) [inline ddl/ddl_test.go:20]: fmt

**Review by hawkingrei — APPROVED** [review]

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**